Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -191,6 +206,7 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
         imageCollectionView.dataSource = self
         imageCollectionView.delegate = self
         imageCollectionView.pagingEnabled = true
+        imageCollectionView.backgroundColor = backgroundColor
```

Comment:
Try to remove [temporal coupling](http://blog.ploeh.dk/2011/05/24/DesignSmellTemporalCoupling/) as much as possible from your code. If someone changes the background color *after* `setupCollectionView` has been called this won't work. On top of that there is no test to demonstrate this and there's no indication in the variable name or anything that this must be set before the view is actually instantiated. Look at [my other comment](https://github.com/Inspirato/SwiftPhotoGallery/pull/3/files#r39881436) on how to address this.